docs/content/guides/dagster/how-assets-relate-to-ops-and-graphs.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Understanding how software-defined assets relate to ops and graphs | Dagster Docs
+description: Dagster’s main abstraction for building data pipelines is the software-defined asset. However, Dagster also has ops and graphs. How do these relate? When should you use one over the other?
+---
+
+# Understanding how software-defined assets relate to ops and graphs
+
+Dagster’s main abstraction for building data pipelines is the _software-defined asset_. However, Dagster also has abstractions called _ops_ and _graphs_.
+
+If you're not sure which one to use, this guide is for you. In this guide, we'll cover:
+
+- When you should assets or ops and graphs
+- How assets, ops, and graphs relate to each other
+- Why Dagster offers all three abstractions
+
+---
+
+## When should I use assets or ops and graphs?
+
+Dagster is mainly used to build data pipelines, and most data pipelines can be expressed in Dagster as sets of [software-defined assets](/concepts/assets/software-defined-assets). If you’re a new Dagster user **and** your goal is to build a data pipeline, we recommend starting with software-defined assets and not worrying about ops or graphs. This is because most of the code you’ll be writing will directly relate to producing data assets.
+
+However, there are some situations where you want to run code without thinking about data assets that the code is producing. In these cases, it’s appropriate to use [ops](/concepts/ops-jobs-graphs/ops) and [graphs](/concepts/ops-jobs-graphs/graphs). For example:
+
+- [You're not building a data pipeline](#situation-1-youre-not-building-a-data-pipeline)
+- [You want to break an asset into multiple steps](#situation-2-you-want-to-break-an-asset-into-multiple-steps)
+- [You're anchored in task-based workflows](#situation-3-youre-anchored-in-task-based-workflows)
+
+### Situation 1: You’re not building a data pipeline
+
+Let's say you want to schedule a workflow where the goal is **not** to keep a set of data assets up-to-date. It might do something like:
```

Comment:
@erinkcochran87 nitpick: I'm not a fan of "Let's say" in docs where we're not otherwise using first person